python/graphstorm/config/argument.py

```diff
@@ -2122,6 +2179,10 @@ def _add_lm_model_args(parser):
     group.add_argument("--freeze-lm-encoder-epochs", type=int, default=argparse.SUPPRESS,
             help="Before fine-tuning LM model, how many epochs we will take "
                  "to warmup a GNN model")
+    group.add_argument("--max-global-step", type=int, default=argparse.SUPPRESS,
+                       help="The maximum of global step for each node type for distillation")
```

Comment:
The maximal number of steps. what does "global" mean here?

Reply:
meaning the steps across all trainers.

Reply:
but having more trainers only change the batch size. it doesn't change the number of iterations.

Reply:
addressed by new commit.